index.html

```diff
@@ -64,15 +63,15 @@ <h1>Bikes for Refugees</h1>
               Providing donated bikes and accessories to refugees and asylum
               seekers in Scotland.
             </p>
-            <button>Donate a bike today</button>
-            <button>Volunteer</button>
+            <button class="donate-button">Donate a bike today</button>
+            <button class="volunteer-button">Volunteer</button>
```

Comment:
On hover on this button, nothing happens, the background should become orange and the text become white
![Screenshot 2023-09-27 at 10 36 29](https://github.com/CodeYourFuture/bikes-for-refugees/assets/13808341/fc5159e6-589a-4f2e-9a23-84be303c23a4)